app/code/core/Mage/Adminhtml/Block/System/Config/Form.php

```diff
@@ -423,6 +424,9 @@ public function initFields($fieldset, $group, $section, $fieldPrefix = '', $labe
                     }
 
                     $sourceModel = Mage::getSingleton($factoryName);
+                    if (!$sourceModel) {
+                        Mage::throwException("Entity type model '{$factoryName}' is not found");
```

Comment:
Could we force phpstan to "trust" the usage by adding a `/** @var */` line above the declaration? It's a bit too much to add this checks everywhere

Reply:
I think the same, it is too much but...

I'll try adding a `/** @var */` and see it works

Reply:
I did it other cases (in this PR) but in this specific one I'd leave it like this, what do you think?

Reply:
Sorry for the delay. Throwing an exception could help with debugging a typo, but the message can be clarified a bit (Source model instead of entity type).